src/main/java/ogc/rs/database/DatabaseServiceImpl.java

```diff
@@ -405,6 +405,53 @@ public Future<JsonObject> getStacCollection(String collectionId) {
                     }));
     return result.future();
   }
+
+  @Override
+  public Future<List<JsonObject>> getStacItems(String collectionId) {
+    Promise<List<JsonObject>> result = Promise.promise();
+    Collector<Row, ?, List<JsonObject>> collector =
+        Collectors.mapping(Row::toJson, Collectors.toList());
+    String getItemsQuery = String.format("select %1$s.id, st_asgeojson(%1$s.geom), %1$s.bbox, %1$s.properties" +
+        ", jsonb_agg((row_to_json(stac_item_assets.*)::jsonb-'stac_item_id')) as assets, 'Feature' as type, %1$s as" +
+            " collection from %1$s join stac_item_assets on %1$s.id=stac_item_assets.item_id" +
+            " group by %1$s.id, %1$s.geometry, %1$s.bbox, %1$s.properties limit 10", collectionId);
```

Comment:
Remove the `LIMIT 10` from the query if no pagination is being implemented for this